### DISCUSSION TITLE
How can I make my commands subscriber-specific?

### DISCUSSION THREAD

**Discussion by Tqeslib** [Sehelitar/Kick.bot#65] [Q&A]
Hello, I did run almost any command. But I don't know how to make my commands just for subs. Can someone help?

**Comment by Tqeslib** [comment]
I forgot to add my second question. Can I shorten the duration of some commands specifically for subscribers? For example, if a follower waits 10 seconds, the subscriber should wait 5 seconds.

**Comment by Sehelitar** [comment]
Hey! 👋 
You can make any command limited to mods, VIPs or subs by using the ``Permissions`` tab. An empty ``Allowed`` column means anyone can run it.
<img width="550" alt="image" src="https://github.com/user-attachments/assets/ae3c33b8-69d1-4056-a735-f4d417626a63">

I don't understand what you mean when you say "shorten the duration of some commands". Do you mean the command's cooldown? Or is this related to your action?
